src/components/05_pages/NodeAddForm/index.js

```diff
@@ -38,19 +46,27 @@ class NodeAddForm extends React.Component {
   };
 
   render() {
-    return (
-      this.props.schema && (
+    let result = null;
+    if (this.props.schema) {
```

Comment:
This needs to check for the existence of the current user

Reply:
@dawehner Since this is `NodeAdd` would there ever be an existing user?

Reply:
Maybe that, but also because otherwise the setting of the default user doesn't work